src/Resources/contao/library/Contao/Template.php

```diff
@@ -265,7 +265,7 @@ public function output()
 
 		header('Vary: User-Agent', false);
 		header('Content-Type: ' . $this->strContentType . '; charset=' . \Config::get('characterSet'));
-
+		 // FIXME: would be cool to throw a response exception here but sadly the hook is in the way.
```

Comment:
Ok, I will demote it to a plain comment. I simply wanted to note why we did not use a response exception here.